moveit_core/CMakeLists.txt

```diff
@@ -21,8 +21,19 @@ endif()
 
 find_package(Boost REQUIRED system filesystem date_time thread iostreams)
 find_package(Eigen3 REQUIRED)
- # TODO: Move collision detection into separate packages
-find_package(Bullet REQUIRED)
+
+# TODO: Move collision detection into separate packages
+find_package(Bullet CONFIG)
+if(BULLET_FOUND AND BULLET_VERSION_STRING VERSION_GREATER "2.86")
```

Comment:
@j-petit can you add TODO explaining when this version issue can be removed? i.e. when we drop support for Kinetic

Reply:
added.